tests/test_training/test_lr_scheduler.py

```diff
@@ -0,0 +1,20 @@
+"""Tests for LR schedulers."""
+
+import pytest
+from class_resolver import HintOrType, OptionalKwargs
+from torch.optim import lr_scheduler
+
+from pykeen.pipeline import pipeline
+
+
+@pytest.mark.parametrize("cls, kwargs", [("CosineAnnealingWarmRestarts", {"T_0": 10})])
```

Comment:
can we add a few more quick test cases, like no regularize, empty dict, etc?

Reply:
To test them in combination with this LR schedule? For general pipeline tests, there should also be something in [test_pipeline](https://github.com/pykeen/pykeen/blob/ac85bef34de3070090012035a2170deddcfb307e/tests/test_pipeline.py).